### DISCUSSION TITLE
Best App ever for PUBG Lovers 🤩

### DISCUSSION THREAD

**Discussion by code-fanatic** [MohamedKVIP/MK-PUBG-Mobile-Tool#3] [Show and tell]
Your work is truly amazing really Appreciate 😍

**Comment by Unknown461** [comment]
Can You Please Add Keymapping Fix Option Also For Pubg Kr:(

**Reply by code-fanatic** [reply]
I have pubg kr keymapping. i am mk's friend but let me contact him first

**Reply by code-fanatic** [reply]
@MohamedKVIP check your discord username is chilling_nightmare my old account got suspended

**Reply by Unknown461** [reply]
Ok🥰And Thanks For Reply

**Reply by MohamedKVIP** [reply]
working on it
